### PR TITLE
Added "git config --add safe.directory"

### PR DESCRIPTION
**Pull request by jdownie**
In my k3s environment the onlyoffice installation was failing because the bare repo was not marked as safe. This addition overcame the issue for me.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN npm install --production \
 # Create actual CryptPad image
 FROM node:lts-slim
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt update && apt install -y git rdfind && rm -rf /var/lib/apt/lists/*
+RUN apt update && apt install -y git rdfind procps net-tools curl && rm -rf /var/lib/apt/lists/*
```
[the remark this answers is not in the document]
Comment (jdownie) [reply Dockerfile:25]: Yeah, fair enough. That shouldn't have been included in my pull request. I'll take that out.

File: install-onlyoffice.sh [hunk 2]
```diff
@@ -117,6 +117,8 @@ ensure_oo_is_downloaded () {
 		echo "Downloading OnlyOffice..."
 		git clone --bare https://github.com/cryptpad/onlyoffice-builds.git "$BUILDS_DIR"
 	fi
+  git config --global --add safe.directory /cryptpad/onlyoffice-conf/onlyoffice-builds.git
```
[the remark this answers is not in the document]
Comment (jdownie) [reply install-onlyoffice.sh]: Yep, will do.

**Comment by jdownie** [comment]
Hi @wginolas, is that roughly what you had in mind? I more or less just ripped off what you had already done for `--accept-license` :wink: .

File: Dockerfile [hunk 3]
```diff
@@ -22,6 +22,7 @@ RUN npm install --production \
 # Create actual CryptPad image
 FROM node:lts-slim
 ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y git rdfind && rm -rf /var/lib/apt/lists/*
```
Comment (wginolas) [inline Dockerfile:25]: There already is an `apt install` line further below. This one can be removed.

**Review by wginolas — CHANGES_REQUESTED** [review]
Thanks for the additional command line arg. I noticed that there are two `apt install` lines in the Dockerfile now. One can be removed.

File: Dockerfile (hunk 3 above)
Comment (jdownie) [reply Dockerfile]: Yeah, it's gone.

**Review by wginolas — APPROVED** [review]
Thank you!

**Pull request merged**